### PR TITLE
[RTI-7763] Apply latest linting and formatting

### PR DESCRIPTION
**Pull request by elbrujohalcon**

### CODE REVIEW

File: src/dynamodb.erl [hunk 1]
```diff
@@ -1,58 +1,75 @@
...
 call(Credentials, Zone, Target, RFCDate, Body) ->
     call(Credentials, Zone, Target, RFCDate, Body, 1000).
 
-
 -spec submit(endpoint(), headers(), iolist(), integer()) -> result().
 submit(Endpoint, Headers, Body, Timeout) ->
     %io:format("Request:~nHeaders:~p~nBody:~n~p~n~n", [Headers, iolist_to_binary(Body)]),
-    case lhttpc:request(Endpoint, "POST", Headers, Body, Timeout, [{max_connections, 10000}]) of
+    case lhttpc:request(Endpoint, "POST", Headers, Body, Timeout, [{max_connections, 10000}])
+        of
```
Comment (raz-adroll) [inline src/dynamodb.erl:72]: This of in a single line looks too weird.

File: src/dmochijson2.erl [hunk 2]
```diff
@@ -461,17 +466,20 @@ tokenize_string(B, S=#decoder{offset=O}, Acc) ->
...
+        <<_:O/binary, C1, C2, _/binary>> when C1 >= 194, C1 =< 223, C2 >= 128, C2 =< 191 ->
             tokenize_string(B, ?ADV_COL(S, 2), [C2, C1 | Acc]);
-        <<_:O/binary, C1, C2, C3, _/binary>> when C1 >= 224, C1 =< 239,
-                C2 >= 128, C2 =< 191,
-                C3 >= 128, C3 =< 191 ->
+        <<_:O/binary, C1, C2, C3, _/binary>>
+            when C1 >= 224, C1 =< 239, C2 >= 128, C2 =< 191, C3 >= 128, C3 =< 191 ->
             tokenize_string(B, ?ADV_COL(S, 3), [C3, C2, C1 | Acc]);
-        <<_:O/binary, C1, C2, C3, C4, _/binary>> when C1 >= 240, C1 =< 244,
-                C2 >= 128, C2 =< 191,
```
Comment (raz-adroll) [inline src/dmochijson2.erl:472]: I won't lie this one looks better because it keeps the original intention (checking ranges) more readable.

**Review by raz-adroll — APPROVED** [review]

File: src/dmochijson2.erl (hunk 2 above)
Comment (elbrujohalcon) [reply src/dmochijson2.erl:472]: I _fixed_ it in an extremely dumb way… 🙈

Comment (elbrujohalcon) [reply src/dmochijson2.erl:472]: I think it's fine since we're talking about `dmochijson2`, a JSON-parsing module stolen from mochiweb… 🤦

**Pull request merged**